### ISSUE TITLE
[Keyboard Navigation - Data Factories - ADF Editor Tree View] Focus indicator is not visible on the Selected item at Editor Tree View Screen.

### ISSUE THREAD

**Issue by sonimishra** [Microsoft/monaco-editor#714]
Blade Name : Data Factories - ADF Editor Tree View

User Experience:
If Focus indicator is not visible on the selected items then User with visual and motion disabilities will get confuse to navigate through the blade.
 
Test Environment:
OS: Win 10 RS3
Build: 16299.15
Platform: Edge, Firefox
Screen Reader: Narrator, NVDA
 
Repro Steps:
1-Navigate to https://portal.azure.com/ and login using valid credentials.
2-Navigate to More Services option and select "data Factory" option.
3-Navigate to various Data factories listed and select anyone
4-Navigate to the Actions section lying on the blade opened and select "Author and Deploy" option.
5-Navigate to New Pipeline option lying on the blade opened and select it then draft item will appear in tree view which is in selected state.
6-Navigate to the selected item and verify the issue.
 
Actual Result:
Focus indicator is not visible on the Selected item at Editor Tree View Screen.
 
Expected result:
Focus indicator should be visible on the Selected item like Draft at Editor Tree View Screen.
 
MAS Reference:
**MAS 2.4.7 - Focus Visible (05A)**
![mas2 4 7_datafactories_focusnotvisibleonalreadyselecttreeitem](https://user-images.githubusercontent.com/30456910/35509079-c7565576-0518-11e8-9411-95405c909c5b.png)

**Comment by sonimishra** [comment]
Azure Data Factory Bug Reference Id : BUG 1826256
https://msazure.visualstudio.com/One/_workitems?id=1826256&_a=edit

**Comment by egamma** [comment]
From the screenshot it looks this issue is out side of the monaco editor, please report the issue against the corresponding azure component.

**Comment by sonimishra** [comment]
Thanks for updated, bug has been logged for Azure portal
